### ISSUE TITLE
Can't install via terminal 

### ISSUE THREAD

**Issue by waglangmath** [daniruiz/flat-remix-gnome#66]
root@Linux:/etc/apt/sources.list.d# add-apt-repository ppa:daniruiz/flat-remix
Cannot add PPA: 'ppa:~daniruiz/ubuntu/flat-remix'.
ERROR: '~daniruiz' user or team does not exist.

**Comment by daniruiz** [comment]
Which version of Ubuntu are you using?

**Comment by waglangmath** [comment]
@daniruiz stretch sir

**Comment by waglangmath** [comment]
sorry sorry, please see ubuntu details below.

Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.5 LTS
Release:	16.04
Codename:	xenial

**Comment by daniruiz** [comment]
Well... Ubuntu 16.04 is 3 years old now and still used Unity Desktop which wasn't replace for GNOME desktop until Ubuntu 17.10. That's why it doesn't work for you. But you can easily update your linux to a much newer version :smile:

**Comment by waglangmath** [comment]
root@Linux:~# lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.2 LTS
Release:	18.04
Codename:	bionic

Upgraded to the latest version,
does the flat-remix run on 17 only?
:(

**Comment by waglangmath** [comment]
my error is actually ERROR: '~daniruiz' user or team does not exist.

**Comment by daniruiz** [comment]
It should work since ubuntu 17.10, I have it working on ubuntu 18.04 and ubuntu 18.10 right now. Try to remove the ppa and then reinstall.
Bear in mind ubuntu disables all third's PPAs after a release upgrade

```bash
sudo add-apt-repository --remove ppa:daniruiz/flat-remix
sudo add-apt-repository ppa:daniruiz/flat-remix
sudo apt update
sudo apt install flat-remix-gnome
```

**Comment by daniruiz** [comment]
![image](https://user-images.githubusercontent.com/12565871/54827677-ddc3d800-4cb2-11e9-86d9-48196919188a.png)
https://launchpad.net/~daniruiz/+archive/ubuntu/flat-remix
As you can see the package is available for ubuntu 18.04 (Bionic Beaver). No idea why it doesn't work for you :sweat:
